packages/design-system/src/components/IdleTimeout/IdleTimeout.test.tsx

```diff
@@ -189,15 +189,15 @@ describe('Idle Timeout', () => {
     const { getByRole } = renderIdleTimeout({ formatMessage, timeToWarning: 2 });
     showWarning(MOCK_START_TIME + 2 * 60000);
     const dialogBodyText = getByRole('main');
-    expect(dialogBodyText.firstChild.textContent).toEqual('Your session will end in 3.');
+    expect(dialogBodyText.firstChild.textContent).toEqual('Your session will end in 3.'); // eslint-disable-line
```

Comment:
I think all of these linting disables should removed personally, I would rather have the linter keep us honest and have us resolve the issues the 'correct' way sometime in the future, than brush these under the rug until who-knows-when..

Reply:
Reposting here from Slack, but we discovered some of our linting rules (no node queries and containers) weren't absolute no-nos when working with RTL, so I've updated our ESLint config to reflect that. As a result, the linting disable comments have been removed.